### ISSUE TITLE
🐞 Bug report: [form widget] Accessibility issue: Errors are not readable by screen readers

### ISSUE THREAD

**Issue by Mulli** [elementor/elementor#17874]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When there is an error, the message is displayed as tooltip that is not readable by screen reader.

An option to have simple `<span>error message</span>` can close this issue

### Steps to reproduce

Any Elementor form with a field of any type that can trigger an error

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Any

**Comment by askwpgirl** [comment]
@nicholaszein  - We are talking about Elementor Accessibility at an upcoming Meetup. Is there an ETA for Elementor Forms accessibility improvements? Thanks!
